Finish the AJAX Request/app.js

```diff
@@ -0,0 +1,8 @@
+var request = new XMLHttpRequest();
```

Comment:
Always use const when the value doesn't change.

Reply:
Thanks, @lycoangelo, This is now updated.